src/js/api/urls.js

```diff
@@ -34,6 +34,8 @@ export const STOCK_TRANSFER_REMOVE_ALL_ITEMS = id => `${STOCK_TRANSFER_BY_ID(id)
 
 // INVOICE
 export const INVOICE_API = `${API}/invoices`;
+export const INVOICE_ITEMS = id => `${INVOICE_API}/${id}/items`;
+export const REMOVE_INVOICE_ITEM = id => `${INVOICE_API}/${id}/removeItem`;
```

Comment:
Maybe it will be worth creating a ticket for a refactoring of part of our URLs. I mean that we should avoid RPC style endpoint and basically in this case it should be enough to remove `/removeItem` because we are already sending a delete request for a specific resource.
I am interested in the opinions of other developers @kchelstowski @awalkowiak

Reply:
I think if we wanted to refactor it, we'd have to have a separate `InvoiceItemApiController` to handle that, because if we were about to remove `removeItem`, we'd have:
```js
/api/invoices/:id
```
that in my opinion when called with `DELETE`, should delete an Invoice, not an InvoiceItem

Reply:
@kchelstowski is correct. 

We could handle this in a Restful way by adding an invoitceItems subresource to the Invoice API. 
```
/api/invoices/:invoiceId/invoiceItems/:invoiceItemId
```
However, I think we can leave this refactoring for later when we actually have the API squared away.